04-tidy.Rmd

```diff
@@ -326,31 +384,40 @@ We'll see in Chapter \@ref(wrangling) how we could use the `mutate()` function t
 ```
 
 **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Convert the `dem_score` data frame into
-a tidy data frame and assign the name of `dem_tidy` to the resulting long-formatted data frame.
+a tidy data frame and assign the name of `dem_score_tidy` to the resulting long-formatted data frame.
 
 **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Read in the life expectancy data stored at http://ismayc.github.io/le_mess.csv and convert it to a tidy data frame. 
 
-```{asis lc4-1solutions, include=show_solutions('4-2')}
-**Learning Check Solutions**  
-
-**`r paste0("(LC", chap, ".", (lc - 1), ")")`** `dem_tidy <- gather(data = dem_score, key = year, value = democracy_score, - country)`
-
-**`r paste0("(LC", chap, ".", (lc), ")")`**
-`life_expectancy <- read_csv('http://ismayc.github.io/le_mess.csv')`  
-
-`life_ex_tidy <- gather(data = life_expectancy, key = year, value = life_expectancy, -country)`
 
+```{asis lc4-3solutions, include=show_solutions('4-3')}
+**Learning Check Solutions**  
 
+**`r paste0("(LC", chap, ".", (lc - 1), ")")`** Running the following in the console  
 ```
-
-```{asis lc4-2solutions, include=show_solutions('4-2'), echo=show_solutions('4-2')}
-`dem_tidy <- gather(data = dem_score, key = year, value = democracy_score, - country)`
-
-`life_expectancy <- read_csv('http://ismayc.github.io/le_mess.csv')`  
-
-`life_ex_tidy <- gather(data = life_expectancy, key = year, value = life_expectancy, -country)`
+```{r lc4-3solutions-2, include=show_solutions('4-3')}
+dem_score_tidy <- gather(data = dem_score, key = year, value = democracy_score, - country)
+```
+```{asis lc4-3solutions-3, include=show_solutions('4-3')}
+Let's now compare the `dem_score` and `dem_score_tidy`. `dem_score` has democracy score information for each year in columns, whereas in `dem_score_tidy` there are explicit variables `year` and `democracy_score`. While both representations of the data contain the same information, we can only use `ggplot()` to create plots using the `dem_score_tidy` data frame.
+```
+```{r lc4-3solutions-4, include=show_solutions('4-3')}
+dem_score
+dem_score_tidy
+```
+```{asis lc4-3solutions-5, include=show_solutions('4-3')}
+**`r paste0("(LC", chap, ".", (lc - 1), ")")`** The code is similar
+```
+```{r lc4-3solutions-6, include=show_solutions('4-3'), echo=show_solutions('4-3'), message=FALSE, warning=FALSE}
+life_expectancy <- read_csv('http://ismayc.github.io/le_mess.csv')
```

Comment:
We'll have to make sure to remember to move these data sets all over to moderndive.com

Reply:
I'll create a new issue so we can quickly merge this PR.